### ISSUE TITLE
[bitnami/postgresql] extraEnv not applied to statefulset-slaves

### ISSUE THREAD

**Issue by pakomp** [bitnami/charts#3086]
**Which chart**:
bitnami/postgresql/8.10.14

**Describe the bug**
I'm trying to activate tls support added on the recent postgres image, by using extraEnv to set 

- POSTGRESQL_ENABLE_TLS
- POSTGRESQL_TLS_CERT_FILE
- POSTGRESQL_TLS_KEY_FILE

but extraEnv only gets added to the master in
charts/bitnami/postgresql/templates/statefulset.yaml
while charts/bitnami/postgresql/templates/statefulset-slaves.yaml doesnt look for this value
so the read replicas wont support tls connections, and I cant find any other way to add it to the slaves.

**To Reproduce**
Add
```
extraEnv:
  - name: POSTGRESQL_ENABLE_TLS
    value: "yes"
  - name: POSTGRESQL_TLS_CERT_FILE
    value: /opt/bitnami/postgresql/cert/tls.crt
  - name: POSTGRESQL_TLS_KEY_FILE
    value: /opt/bitnami/postgresql/cert/tls.key
```
to values.yaml

**Expected behavior**
I expect extraEnv to apply to all servers, if its a master only setting I would expect it to be definied under the master key like how extraVolumes works.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"clean", GoVersion:"go1.13.12"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.0", GitCommit:"70132b0f130acc0bed193d9ba59dd186f0e634cf", GitTreeState:"clean", BuildDate:"2019-12-07T21:20:10Z", GoVersion:"go1.13.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-13T11:13:49Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
If making extraEnv apply to slaves would be a breaking change, maybe adding extraEnv support in the master/slave keys could be none breaking change.

**Comment by joancafom** [comment]
Hi @pakomp ,

Good news, I have implemented a PR #3057 including TLS support to this Chart. It is being reviewed and soon it will be merged. There you can find the changes I have performed in order to have it up and running.

Regarding the `extraEnv`, it seems to me you are right, we currently are only supporting this parameter on the master statefulset. We may implement these changes, as they are a good feature to have. However, to speed up the process and if you like and are up to, you can implement a PR with the necessary changes and we will be more than happy to review them 😁

Thanks!

**Comment by joancafom** [comment]
Hi again @pakomp 

Just to let you know that PR #3057 has been successfully merged! Now the Chart comes with TLS support.

Best regards.
